src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -3,8 +3,11 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 #---------------------------------------------------------------------------------------------
 
+import platform
 import random
 import string
+#pylint: disable=import-error
+from six.moves.urllib.request import urlretrieve
```

Comment:
You probably should add the suppression at the same line ; otherwise the suppression will apply on all the imports in the rest of the file